### DISCUSSION TITLE
HMAC plugin should follow the updated Spec

### DISCUSSION THREAD

**Discussion by ahmadnassri** [Kong/kong#7224] [Ideas and feature requests]
[v5](https://tools.ietf.org/html/draft-cavage-http-signatures-05) is the latest, current implementation deviates from original and references [v00](https://tools.ietf.org/html/draft-cavage-http-signatures-00)

**Comment by BrianHutchison** [comment]
+1

**Comment by peter-evans** [comment]
The latest seems to be [v6](https://tools.ietf.org/html/draft-cavage-http-signatures-06) now.

**Comment by bungle** [comment]
Spec is now updated to [v10](https://tools.ietf.org/html/draft-cavage-http-signatures-10).
